odoo/addons/base/models/ir_attachment.py

```diff
@@ -181,19 +181,25 @@ def _gc_file_store(self):
             cr.rollback()
             return False
 
-        self._gc_file_store_unsafe()
+        done, remaining = self._gc_file_store_unsafe()
 
         # commit to release the lock
         cr.commit()
 
-    def _gc_file_store_unsafe(self):
+        return done, remaining
```

Comment:
That function has now a return type: `None | Literal[False] | tuple[int, int]`
Can you just align it all to have one return type?